test/jenkins_tests/multi_node_tests/many_drivers_test.py

```diff
@@ -44,7 +44,7 @@ def driver(redis_address, driver_index):
     for i in range(driver_index - max_concurrent_drivers + 1):
         _wait_for_event("DRIVER_{}_DONE".format(i), redis_address)
 
-    def try_to_create_actor(actor_class, timeout=100):
+    def try_to_create_actor(actor_class, timeout=500):
```

Comment:
For some reason, this test seems to fail unless we increase the timeout here. I still don't see why this PR affects this test.

In a follow up PR, I'll try releasing GPU resources as soon as an actor exits and see if that speeds up this test.

Another possibility is that the monitor is the bottleneck because it has to process so many drivers that are exiting.